fabtests/functional/atomic_verify.c

```diff
@@ -0,0 +1,214 @@
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+
+#include <rdma/fi_errno.h>
+#include <rdma/fi_atomic.h>
+
+#include "shared.h"
+#include "hmem.h"
+#include "rdma/fi_domain.h"
+
+int do_verification = 1;
```

Comment:
This needs to be made into an option (command line I suppose) because verification will impact performance, so we don't want to always hardcode it on.